strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/configuration/ConfigurationManagerTest.java

```diff
@@ -47,11 +52,7 @@
 public class ConfigurationManagerTest
 {
 
-    private static final String TEST_CLASSES = "target/test-classes";
-
-    private static final String CONFIGURATION_BASEDIR = TEST_CLASSES + "/yaml";
-
-    private static final String CONFIGURATION_OUTPUT_FILE = CONFIGURATION_BASEDIR + "/strongbox-saved-cm.yaml";
+    private static final Path CONFIGURATION_BASEDIR = Paths.get("target", "test-classes", "yaml");
```

Comment:
@ptirador can you please try to use `propertiesBooter.getVaultDirectory()` instead of this?